pkgs/servers/flaresolverr/default.nix

```
@@ -0,0 +1,50 @@
+{ lib, stdenv, python3, chromium, xvfb-run, xorgserver, makeWrapper, chromedriver, fetchFromGitHub }:
+
+let
+  python_env = python3.withPackages
+    (p: with p; [ bottle waitress selenium func-timeout requests websockets xvfbwrapper webtest ]);
+in
+stdenv.mkDerivation rec {
+  pname = "flaresolverr";
+  version = "3.0.2";
+
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-zpeJf1CaQ4bsncZz44sH+tFKddYrZf7YdNYL50d9GA4=";
+  };
+
+  buildInputs = [ makeWrapper ];
+  nativeBuildInputs = [ chromedriver ];
+  patches = [ ./chromedriver_path.patch ];
+
+  postPatch = ''
+    substituteInPlace src/utils.py \
+    --replace "NIX_STORE_DRIVER_PATH" "${chromedriver}/bin/chromedriver"
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/share
+    cp -r src $out/share
+    cp package.json $out/share
```

```suggestion
    cp -r src $out/share/flaresolverr
    cp package.json $out/share/flaresolverr
```
otherwise we might run into conflicts with other packages when installing system wide